### PR TITLE
OSDOCS-12045: Updated the Network config. parameters doc to include OSP

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16+

Issue:
[OSDOCS-12045](https://issues.redhat.com/browse/OSDOCS-12045)

Link to docs preview:
* [Network configuration parameters - OpenStack](https://82822--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installation-config-parameters-openstack#installation-configuration-parameters-network_installation-config-parameters-openstack)
* [Network configuration parameters - vSphere](https://82822--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installation-config-parameters-vsphere#installation-configuration-parameters-network_installation-config-parameters-vsphere)

- [x] SME has approved this change.
- [x] QE has approved this change (itbrown).

Add. resources:
* https://github.com/openshift/openshift-docs/pull/79422
* [GitLab example](https://gitlab.cee.redhat.com/itzikb/rfes/-/blob/master/ipv6-dualstack/install-config.yaml?ref_type=heads)

### CODE REVIEW

**Review by maxwelldb — APPROVED** [review]

**Review by maxwelldb — APPROVED** [review]

**Comment by itzikb-redhat** [comment]
/lgtm
@MaysaMacedo Can you please take a look?

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -182,16 +182,11 @@ endif::ibm-power-vs[]
 
 You can customize your installation configuration based on the requirements of your existing network infrastructure. For example, you can expand the IP address block for the cluster network or provide different IP address blocks than the defaults.
 
-// OSDOCS-1640 - IPv4/IPv6 dual-stack bare metal only
-// But only for installer-provisioned
-// https://bugzilla.redhat.com/show_bug.cgi?id=2020416
-// Once BM UPI supports dual-stack, uncomment all the following conditionals and blocks
-
-ifndef::agent,bare,ibm-power,ibm-z,vsphere[]
+ifndef::agent,bare,ibm-power,ibm-z,vsphere,osp[]
```
Comment (dfitzmau) [inline modules/installation-configuration-parameters.adoc:185]: The ifndef block means that the `Only IPv4 addresses are supported.` statement won't be included in the OpenStack docs.

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -182,16 +182,11 @@ endif::ibm-power-vs[]
...
 Only IPv4 addresses are supported.
-endif::agent,bare,ibm-power,ibm-z,vsphere[]
+endif::agent,bare,ibm-power,ibm-z,vsphere,osp[]
 
-ifdef::agent,bare,ibm-power,ibm-z,vsphere[]
+ifdef::agent,bare,ibm-power,ibm-z,vsphere,osp[]
```
Comment (dfitzmau) [inline modules/installation-configuration-parameters.adoc:189]: The ifdef block means that the content will be included in the OpenStack docs.

**Pull request merged**